app/views/RegisterView.js

```diff
@@ -119,13 +119,20 @@ class RegisterView extends React.Component {
 		const {
 			name, email, password, username, customFields
 		} = this.state;
-		const { loginRequest } = this.props;
 
 		try {
 			await RocketChat.register({
 				name, email, pass: password, username, ...customFields
 			});
-			await loginRequest({ user: email, password });
+			const { navigation } = this.props;
```

Comment:
There's a setting on the server enforcing email verification.
You should do this only if that setting is enabled.